WorkloadManagementSystem/scripts/dirac-admin-get-pilotslogging.py

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env python
+"""
+  Get Pilots Logging for specific Pilot UUID or Job ID.
+"""
+__RCSID__ = "$Id$"
+
+import sys
+from DIRAC import S_OK
+from DIRAC.Core.Base import Script
+from DIRAC.WorkloadManagementSystem.Client.PilotsLoggingClient import PilotsLoggingClient
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB import PilotAgentsDB
+from DIRAC.Core.Utilities.PrettyPrint import printTable
+
+uuid = None
+jobid = None
+
+def setUUID( optVal ):
+  """
+  Set UUID from arguments
+  """
+  global uuid
+  uuid = optVal
+  return S_OK()
+
+def setJobID( optVal ):
+  """
+  Set JobID from arguments
+  """
+  global jobid
+  jobid = optVal
+  return S_OK()
+
+Script.registerSwitch( 'u:', 'uuid=', 'get PilotsLogging for given Pilot UUID', setUUID )
+Script.registerSwitch( 'j:', 'jobid=', 'get PilotsLogging for given Job ID', setJobID )
+
+Script.setUsageMessage( '\n'.join( [ __doc__.split( '\n' )[1],
+                                     'Usage:',
+                                     '  %s option value ' % Script.scriptName ] ) )
```

Comment:
Maybe just print a usage message saying that either the JobID or the UUID should be added. But not both.